.circleci/config.yml

```diff
@@ -26,14 +26,14 @@ windows_defaults: &windows_defaults
 slack-fail-notify: &slack-fail-notify
   slack/notify:
     event: fail
-    channel: lumos-alerts
+    channel: team-unify-alerts
```

Comment:
I think team `Unify` might prefer having those notifications directed to the `team-unify-pipeline-info` channel instead the alerting channel 🤔 

@danlucian Do you have a preference on where to direct failed pipeline notifications?